.ci/deps.nltk.sh

```diff
@@ -0,0 +1,17 @@
+set -e
+set -x
+
+
+NLTK_DATA_PATH=~/nltk_data
```

Comment:
I guess I've done this already. Please check `cache_directories` in
circle.yml once.
ᐧ

On Mon, Apr 4, 2016 at 11:06 PM, Mischa Krüger <notifications@github.com>
wrote:

> In .ci/deps.nltk.sh
> <https://github.com/coala-analyzer/coala-bears/pull/315#discussion_r58415067>
> :
>
> > @@ -0,0 +1,17 @@
> > +set -e
> > +set -x
> > +
> > +
> > +NLTK_DATA_PATH=~/nltk_data
>
> don't you need to register this path to the cache?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/315/files/44ad0a60bd0ebd1a734c14f78a7a669e30537549#r58415067>
>